AAC freshness updates

BryanLa opened the pull request:
# Contributor guide pull request

Thanks for contributing to the [Microsoft Learn contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.

## Quality control

- [ ] 1. **Successful build with no warnings**: Review the build status to make sure **all checks are green** (Succeeded).

- [ ] 2. **#Sign-off**: Once the PR is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the PR. Signing off means the document is ready for review and can be published at any time.

## Merge and publish

- All PRs to this repository are manually reviewed and merged.
- Once all feedback on the PR is addressed, we'll merge the PR into the main branch.

## Need help?

- If you have an open PR, tag the PR reviewers in comment with your question: `@carlyrevier, @jehchow`
- Open an issue in this repo and tag the PR reviewers.

stephen-sumner commented:
@BryanLa - really nice! My thoughts were to add pictures of a successful build and "#sign-off". 

What if we don't tell them to delete and just add a Description header. That way they have the directions for the entire PR? Perhaps we state the whole process from there perspective to set the expectation of review and addressing feedback. Maybe:

Thanks for contributing to the [Microsoft Learn contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please enter a brief description and read the instructions for processing your pull request (PR).

## Brief PR description
[ Enter brief description here ]

## Instructions
1. Entire brief description: .... etc. (maybe deduplicate)
2. Have a successful build: .... etc. 
3. Sign off: .... etc. 
4. Wait for review: ... etc. 
5. Address all feedback: ... etc.

BryanLa commented:
> @BryanLa - really nice! My thoughts were to add pictures of a successful build and "#sign-off".
> 
> What if we don't tell them to delete and just add a Description header. That way they have the directions for the entire PR? Perhaps we state the whole process from there perspective to set the expectation of review and addressing feedback.
> 
> Maybe something like:
> 
> "Thanks for contributing to the [Microsoft Learn contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please enter a brief description and read the instructions for processing your pull request (PR).
> 
> ## Brief PR description
> [ Enter brief description here ]
> 
> ## Instructions
> 1. Entire brief description: .... etc. (maybe deduplicate)
> 2. Have a successful build: .... etc.
> 3. Sign off: .... etc.
> 4. Wait for review: ... etc.
> 5. Address all feedback: ... etc. "

Thanks @stephen-sumner !

Was this feedback for the PR template? Or for this PR (ie: the Architecture Center public contributor guide page)? 

If for the latter, check out [the link](https://learn.microsoft.com/contribute/content/#quick-edits-to-documentation) in the "Edit an existing article" section. It links them to the public contributor guide page which includes all of the instructions you cited, as well as screenshots. 

If for the PR template, we should vet this with Chad. He and I had several discussions on the PR template, including taking a very minimalist approach to show only relevant details for PR creation (and tucking detailed instructions in the expand/collapse  section of the template for build info, sign-off, review, etc., minus screenshots but they can certainly be added).

BryanLa commented:
#sign-off
